src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {'signup' in queryParams ?
             <p>
-              <Link
-                to={`/${orgId}/${projectId}/?onboarding=1#welcome`}
+              <a
```

Comment:
I'm adding a #welcome to the url, which triggers the onboarding panel to be opened. hashchange event isn't being triggered with <Link>, nor is the component being remounted. I think it's something to do with React Router, as they use the hash as a fallback.

Changing it to <a /> reloads the whole page, which allows me to open the on-boarding panel onComponentMount.